sql/src/main/java/io/crate/planner/consumer/QualifiedNameCounter.java

```diff
@@ -22,17 +22,27 @@
 
 package io.crate.planner.consumer;
 
-import io.crate.analyze.relations.AnalyzedRelation;
-import io.crate.analyze.relations.AnalyzedRelationVisitor;
-import io.crate.planner.Plan;
+import io.crate.analyze.symbol.DefaultTraversalSymbolVisitor;
+import io.crate.analyze.symbol.Field;
+import io.crate.analyze.symbol.MatchPredicate;
+import io.crate.sql.tree.QualifiedName;
 
-public class RelationPlanningVisitor extends AnalyzedRelationVisitor<ConsumerContext, Plan> {
+import java.util.Set;
+
+public class QualifiedNameCounter extends DefaultTraversalSymbolVisitor<Set<QualifiedName>, Void> {
```

Comment:
It's `Counter` because it's not new but was part of the ManyTableConsumer and I moved it into a separate class - but you're right it doesn't fit and I'll change it.